tests/utils/inputparams/testparams.go

```diff
@@ -23,8 +23,17 @@ import (
 	"os"
 	"strconv"
 	"time"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
```

Comment:
CI is failing due to this import.

```
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x65a700, 0xc4200100f0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/vmware/docker-volume-vsphere/tests/utils/ssh.init()
	/drone/src/github.com/vmware/docker-volume-vsphere/tests/utils/ssh/ssh.go:27 +0x177
github.com/vmware/docker-volume-vsphere/tests/utils/govc.init()
	/drone/src/github.com/vmware/docker-volume-vsphere/tests/utils/govc/cmd.go:68 +0x38
github.com/vmware/docker-volume-vsphere/tests/utils/inputparams.init()
	/drone/src/github.com/vmware/docker-volume-vsphere/tests/utils/inputparams/testparams.go:145 +0x47
github.com/vmware/docker-volume-vsphere/vmdk_plugin/drivers/vmdk/vmdkops_test.init()
	/drone/src/github.com/vmware/docker-volume-vsphere/vmdk_plugin/drivers/vmdk/vmdkops/cmd_test.go:50 +0x33
main.init()
	github.com/vmware/docker-volume-vsphere/vmdk_plugin/drivers/vmdk/vmdkops/_test/_testmain.go:110
```

Proposed solution:
1. move https://github.com/vmware/docker-volume-vsphere/blob/master/tests/utils/ssh/ssh.go#L27 `sshIdentity` parameters to this file 
2. make sure `os.Getenv("SSH_KEY_OPT")` not empty before splitting. 
3. have an util for generating `sshIdentity` parameters and ssh.go should call to inputParams to retrieve it. 

@govint Please let me know if you can address as part of this review otherwise I can send out review to unblock yours.